Report improvements (#522)
desmos-labs/desmos: docs/developers/types/staging/posts/report.md

Inside Desmos, users can flag posts with reports.

## Contained data
Report contains data that allows users to create the most complete report they need.

### `Reasons`

> The reasons of the report which correspond to one or more reasons saved in chain params.

Reviewer: ```suggestion
The reasons of the report which correspond to one or more reasons saved as the module params.
```
